cdci_data_analysis/analysis/parameters.py

```diff
@@ -246,31 +269,33 @@ def set_from_form(self, form, verbose=False):
             logger.error("problem e=%s setting par_name=%s, form=%s",
                          repr(e),
                          par_name,
-                         form,
-                        )
+                         form
+                         )
             raise
 
         if in_dictionary is True:
-            self.set_par(value=v, units=u)
+            form[par_name] = self.set_par(value=v, units=u)
         else:
             # set the default value
             form[par_name] = self.value
+
             if verbose is True:
                 logger.debug('setting par: ', par_name, ' not in dictionary, setting to the default value')
```

Comment:
this is not how logging is to be used. Actually this will raise an exception in logging. I realize it was not added just now, but still.

Reply:
do we still enable somewhere the `verbose` option?

Reply:
yes, well, `logging` is controlled without explict verbose, depends how the app is started. One way in test, another in gunicorn, third in ther https://github.com/oda-hub/dispatcher-app/blob/master/bin/run_osa_cdci_server.py#L66

What's important is to set message levels and logger names.